packages/flutter_tools/lib/src/version.dart

```diff
@@ -10,10 +10,12 @@ import 'package:quiver/time.dart';
 
 import 'base/common.dart';
 import 'base/context.dart';
+import 'base/file_system.dart';
 import 'base/io.dart';
 import 'base/process.dart';
 import 'base/process_manager.dart';
 import 'cache.dart';
+import 'common.dart';
```

Comment:
It's already imported above - in `base/common.dart`

Reply:
Yes, see that now. But `getFlutterRoot()` is in `test/src/common.dart` not `lib/src/base/common.dart`. How do files in `lib` get the root?